proxysql/dockerdir/entrypoint.sh

```diff
@@ -25,16 +25,15 @@ PROXY_CFG=/etc/proxysql/proxysql.cnf
 PROXY_ADMIN_CFG=/etc/proxysql-admin.cnf
 
 set +o xtrace # hide sensitive information
-MYSQL_ROOT_PASSWORD_ESCAPED=$(sed 's/[\*\.\@\&\#\?\!]/\\&/g' <<<"${MYSQL_ROOT_PASSWORD}")
```

Comment:
could we add escaping for monitor and proxyadmin password?

Reply:
added https://github.com/percona/percona-docker/pull/235/commits/81d332e20e6046d09b41d9756fa371387fa74d1e